queue/driver_group_mgo.go

```diff
@@ -396,16 +396,16 @@ func (d *mgoGroupDriver) Next(ctx context.Context) amboy.Job {
 		},
 	}
 
+	timeLimits := bson.M{}
+	now := time.Now()
 	if d.opts.CheckWaitUntil {
-		qd = bson.M{
-			"$and": []bson.M{
-				qd,
-				{"$or": []bson.M{
-					{"time_info.wait_until": bson.M{"$lte": time.Now()}},
-					{"time_info.wait_until": bson.M{"$exists": false}}},
-				},
-			},
-		}
+		timeLimits["time_info.wait_until"] = bson.M{"$lte": now}
```

Comment:
Does this need to check that it's not 0? (similar question a couple more times below too)

Reply:
if it's zero, then it'll be less than now, so I think it's fine.